runtime/Cargo.toml

```diff
@@ -29,7 +29,7 @@ tracing-core = {optional = true, version = '0.1.17'}
 module-amm-rpc-runtime-api = {path = '../pallets/amm/rpc/runtime-api', default-features = false, version = '2.0.0'}
 pallet-amm = {path = '../pallets/amm', default-features = false, version = '2.0.0'}
 pallet-asset-registry = {path = '../pallets/asset-registry', default-features = false, version = '2.0.0'}
-pallet-claims = {path = '../pallets/claims', default-features = false, version = '2.0.0'}
+pallet-claims = {path = '../pallets/claims', default-features = false, version = '2.0.1'}
```

Comment:
We probably don't need a version here right now since we have just one runtime and (planning for two) but it should always be compatible. One less bump? (We can do this in separate issue)

Reply:
the version is not needed in this case, as it is local depedency where you'll have always only 1 version. I already did in another  PR for amm pallet, without version.